### PR TITLE
Add mintlify

### PR DESCRIPTION
**Pull request by arnehuang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ This should be updated fairly regularly. As usual, **pull requests are encourage
 
 ### Generating Docs
 
+* [Mintlify](https://mintlify.com/) - Hosted docs using Markdown and MDX (contributed by [arnehuang](https://github.com/arnehuang)))
```
Comment (matheusfelipeog) [inline README.md:86]: Remove this extra character:

```suggestion
* [Mintlify](https://mintlify.com/) - Hosted docs using Markdown and MDX (contributed by [arnehuang](https://github.com/arnehuang))
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @arnehuang ;)

New additions should be added to the end of the list, please.

Also, can you solve the review below?

**Comment by arnehuang** [comment]
thanks, done

**Review by matheusfelipeog — APPROVED** [review]
Thanks @arnehuang 😄

**Pull request merged**